WcaOnRails/app/models/registration.rb

```diff
@@ -122,8 +122,11 @@ def waiting_list_info
 
   validate :user_can_register_for_competition
   private def user_can_register_for_competition
-    if user&.cannot_register_for_competition_reasons.present?
```

Comment:
Good call. I guess I was trying to save ourselves from calling the same function twice, but I agree it's not worth the extra 3 lines of code.